QuantConnect.TDAmeritrade/TDAmeritradeBrokerage.WebSocket.cs

```diff
@@ -498,35 +499,19 @@ private void OnBestBidAskUpdated(object? sender, BestBidAskUpdatedEventArgs e)
         /// <param name="askSize">The ask price</param>
         private void EmitQuoteTick(Symbol symbol, decimal bidPrice, decimal bidSize, decimal askPrice, decimal askSize)
         {
-            var exchange = GetSymbolExchange(symbol);
-
             _aggregator.Update(new Tick
             {
                 AskPrice = askPrice,
                 BidPrice = bidPrice,
                 Value = (askPrice + bidPrice) / 2m,
-                Time = DateTime.UtcNow.ConvertFromUtc(exchange),
+                Time = DateTime.UtcNow.ConvertFromUtc(TimeZones.NewYork),
                 Symbol = symbol,
                 TickType = TickType.Quote,
                 AskSize = askSize,
                 BidSize = bidSize
             });
         }
 
-        private DateTimeZone GetSymbolExchange(Symbol symbol)
-        {
-            lock (_symbolExchangeTimeZones)
-            {
-                if (!_symbolExchangeTimeZones.TryGetValue(symbol, out var exchangeTimeZone))
-                {
-                    // read the exchange time zone from market-hours-database
-                    exchangeTimeZone = MarketHoursDatabase.FromDataFolder().GetExchangeHours(symbol.ID.Market, symbol, symbol.SecurityType).TimeZone;
-                    _symbolExchangeTimeZones[symbol] = exchangeTimeZone;
-                }
-                return exchangeTimeZone;
-            }
-        }
```

Comment:
Should keep this method and not hard code `TimeZone.NewYork` usage in quotes and trades

Reply:
I came back this method. Also, I unhardcoded timeZone in quote and trade.